tests/EalTest.cpp

```diff
@@ -22,19 +24,6 @@ TEST(PositionInterpTest, LinearInterp) {
   EXPECT_DOUBLE_EQ(-4.5, coordinate[2]);
 }
 
-TEST(PositionInterpTest, SplineInterp) {
-  vector<double> times = {0,  1,  2, 3};
-  vector<vector<double>> data = {{0, 0, 0, 0},
-                                 {0, 1, 2, 3},
-                                 {0, 2, 1, 0}};
-
-  vector<double> coordinate = eal::getPosition(data, times, 0.5, eal::spline);
-
-  ASSERT_EQ(3, coordinate.size());
-  EXPECT_DOUBLE_EQ(0,      coordinate[0]);
-  EXPECT_DOUBLE_EQ(0.5,   coordinate[1]);
-  EXPECT_DOUBLE_EQ(2.8 * 0.5 - 0.8 * 0.125, coordinate[2]);
-}
```

Comment:
I took this test out because we are testing the spline interpolation separately now